opencga-app/app/scripts/azure/arm/appgw/azuredeploy.json

```diff
@@ -23,6 +29,10 @@
                 "description": "DNS prefix used by the app gateway IP"
             }
         },
+        "appGwSslEmail": {
+            "type": "string",
+            "defaultValue": "user@exampledomain.com"

```

Comment:
Not sure this is a good default. Suggest either removing the default, or using something @example.com.

Reply:
Agreed, accidentally left it there when testing. defaultValue removed in 40a78ca